test/test_nestedtensor.py

```diff
@@ -911,6 +911,59 @@ def test_nested_tensor_chunk(self, device, dtype):
         self.assertRaisesRegex(RuntimeError, "derivative for aten::chunk is not implemented",
                                lambda: chunked[0].backward(chunked[0].clone()))
 
+    @dtypes(*floating_types_and_half())
+    def test_nested_tensor_split_with_sizes(self, device, dtype):
+        a = torch.randn(3, 20, device=device, dtype=dtype)
+        b = torch.randn(2, 20, device=device, dtype=dtype)
+        c = torch.randn(1, 20, device=device, dtype=dtype)
+
+        split_sizes = [4, 6, 10]
+        a_splits = a.split_with_sizes(split_sizes, dim=-1)
```

Comment:
Does it only work with dim=-1?

Reply:
I'd just add a test that exercises that failure

Reply:
oh there is one, lemme comment it correctly